inginious/common/tests/test_course_user_settings.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
@nrybowski can you give your opinion on that file ? it still confuses me. I'd rather see it removed from the PR so that we can finally move on.